lib/wallet/test/unit/Cardano/Wallet/DB/LayerSpec.hs

```diff
@@ -1332,7 +1343,8 @@ testCreateMetadataTable ::
 testCreateMetadataTable = withSystemTempFile "db.sql" $ \path _ -> do
     let noop _ = pure ()
         tr = nullTracer
-    withDBLayer @s @k tr defaultFieldValues path dummyTimeInterpreter noop
+    withDBLayer @s @k tr (Just defaultFieldValues)
+        path dummyTimeInterpreter testWid noop
```

Comment:
```suggestion
    withDBOpenFromFile @s @k
        tr (Just defaultFieldValues) path noop
```

As this unit test does not require the presence of a particular walletId, and uses a raw SQL statement, I think it's better to use the weaker `DBOpen` here.